test/users.tests.js

```diff
@@ -125,4 +146,31 @@ describe('users', function () {
       expect(error.username).to.include('uqweu');
     });
   });
-});
+
+  describe.skip('changePassword with username and password', function () {
+    var profile;
+
+    before(function (done) {
+      users.changePassword('john', password, function (err, p) {
```

Comment:
`john` is the user we use to run all tests. Let's change this to another user we will create in AD to test this specifically. I'll provide the user details for the expectations